src/lib/salad/hll_empirical.h

```diff
@@ -0,0 +1,27 @@
+#ifndef HLL_EMPIRICAL_H_INCLUDED
+#define HLL_EMPIRICAL_H_INCLUDED
+
+#include <stdint.h>
+
+/*
+ * HyperLogLog needs a bias correction for small cardianlities.
+ * The bias corrections was found empirically for precsion range
+ * defined by following constants.
+ */
+#define HLL_MIN_PRECISION  6
+#define HLL_MAX_PRECISION  18
+
+/*
+ * Return empirical based interpolated bias correction for the raw_estimation.
+ */
+double
+hll_empirical_bias_correction(uint8_t precision, double raw_estimation);
```

Comment:
Are these two functions enough to operate on HLL? I guess no

Reply:
@Korablev77 what?